### ISSUE TITLE
Menu component page is broken in docs

### ISSUE THREAD

**Issue by lasernite** [vuematerial/vue-material#855]
https://vuematerial.github.io/#/components/menu is pulling up blank. 

```
Vue warn]: Error in render function: "TypeError: Cannot read property 'open' of undefined"

found in

---> <Anonymous>
       <Root>
wr @ vendor.5d5d08d7.js:3
k @ vendor.5d5d08d7.js:1
t._render @ vendor.5d5d08d7.js:2
Et.r @ vendor.5d5d08d7.js:2
To.get @ vendor.5d5d08d7.js:3
To @ vendor.5d5d08d7.js:3
Et @ vendor.5d5d08d7.js:2
Se.$mount @ vendor.5d5d08d7.js:3
init @ vendor.5d5d08d7.js:3
c @ vendor.5d5d08d7.js:2
s @ vendor.5d5d08d7.js:2
(anonymous) @ vendor.5d5d08d7.js:2
t._update @ vendor.5d5d08d7.js:1
Et.r @ vendor.5d5d08d7.js:2
To.get @ vendor.5d5d08d7.js:3
To.run @ vendor.5d5d08d7.js:3
Mt @ vendor.5d5d08d7.js:2
(anonymous) @ vendor.5d5d08d7.js:3
t @ vendor.5d5d08d7.js:3
vendor.5d5d08d7.js:1 TypeError: Cannot read property 'open' of undefined
    at Proxy.render (menu.2d8baa82.js:1)
    at a.t._render (vendor.5d5d08d7.js:2)
    at a.Et.r (vendor.5d5d08d7.js:2)
    at To.get (vendor.5d5d08d7.js:3)
    at new To (vendor.5d5d08d7.js:3)
    at Et (vendor.5d5d08d7.js:2)
    at a.Se.$mount (vendor.5d5d08d7.js:3)
    at init (vendor.5d5d08d7.js:3)
    at c (vendor.5d5d08d7.js:2)
    at s (vendor.5d5d08d7.js:2)```

**Comment by pablohpsilva** [comment]
I'll fix it ASAP. Thank you!

**Comment by pablohpsilva** [comment]
Fixed on #856
